test/observations.jl

```diff
@@ -218,6 +218,37 @@ using NEOs: src_path
         @test source_radar == check_radar
     end
 
+    @testset "NEOCPObject" begin
+        using NEOs: NEOCPObject
+
+        # Fetch current NEOCP list
+        objects = fetch_objects_neocp()
+        @test isa(objects, Vector{NEOCPObject{Float64}})
+        # Pick the first object
+        object = objects[1]
+        # Check NEOCPObject fields
+        @test 0 <= object.score <= 100
+        @test object.date >= Date(2024)
```

Comment:
```suggestion
        @test object.date >= year(today())
```

Reply:
I think something even better would be:
```julia
@test object.date >= today() - Month(3)
```
since we do not expect objects to be so long on the NEOCP.

Reply:
I fully agree; my point was simply to make something automatic, i.e., do not depend on "2024".